### PR TITLE
Improve reference wallet

### PR DESCRIPTION
**Pull request by Juliano1612**
This improves the reference wallet adding a better UI and more features implemented on `wallet-sdk-swift`.

### CODE REVIEW

File: Targets/AppUIKit/Sources/verifier/VerifyDLView.swift [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    @State var success: Bool?
+
+    @Binding var path: NavigationPath
+
+    var body: some View {
+        if success == nil {
+            ScanningComponent(
+                path: $path,
+                scanningParams: Scanning(
+                    subtitle: "Scan the\nback of your driver's license",
```
Comment (w4ll3) [inline Targets/AppUIKit/Sources/verifier/VerifyDLView.swift:17]: can we get rid of the breakline and let Swift UI handle linebreak?

File: Targets/AppUIKit/Sources/verifier/VerifyEAView.swift [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    @Binding var path: NavigationPath
+
+    var body: some View {
+        if success == nil {
+            if stepOneValue == nil {
+                ScanningComponent(
+                    path: $path,
+                    scanningParams: Scanning(
+                        subtitle: "Scan the front of your\nemployment authorization",
```
Comment (w4ll3) [inline Targets/AppUIKit/Sources/verifier/VerifyEAView.swift:19]: same as before for breaklines

File: Targets/AppUIKit/Sources/wallet/ShareableCredentialListItem.swift [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+                    Spacer()
+                }
+                .padding(.vertical, 12)
+                .onTapGesture {
+                    showingQRCode.toggle()
+                }
+                if showingQRCode {
+                    QRSheetView(credentials: credentials)
+
+                    Text("Shares your credential online or \n in-person, wherever accepted.")
```
Comment (w4ll3) [inline Targets/AppUIKit/Sources/wallet/ShareableCredentialListItem.swift:103]: line break

**Review by w4ll3 — APPROVED** [review]
Some comments that might not be relevant, otherwise LGTM

**Comment by sbihel** [comment]
The conflicts are due to #7, but it should be fairly easy to adapt it to your verifier code

File: Targets/AppUIKit/Sources/verifier/VerifyDLView.swift (hunk 1 above)
Comment (Juliano1612) [reply Targets/AppUIKit/Sources/verifier/VerifyDLView.swift:17]: this is just to make it equal to the designs

File: Targets/AppUIKit/Sources/verifier/VerifyEAView.swift (hunk 2 above)
Comment (Juliano1612) [reply Targets/AppUIKit/Sources/verifier/VerifyEAView.swift:19]: this is just to make it equal to the designs

File: Targets/AppUIKit/Sources/wallet/ShareableCredentialListItem.swift (hunk 3 above)
Comment (Juliano1612) [reply Targets/AppUIKit/Sources/wallet/ShareableCredentialListItem.swift:103]: this is just to make it equal to the designs

File: Targets/AppUIKit/Sources/HomeView.swift [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import SwiftUI
+
+struct HomeView: View {
+
+    @Binding var path: NavigationPath
+
+    var body: some View {
+        TabView {
```
Comment (sbihel) [inline Targets/AppUIKit/Sources/HomeView.swift:8]: Is it on purpose that the items don't have icons? Makes it hard to tap on them

File: project.yml [hunk 5]
```diff
@@ -46,17 +46,33 @@ targets:
...
+          - Inter-Italic.ttf
+          - Inter-Light.ttf
+          - Inter-LightItalic.ttf
+          - Inter-Medium.ttf
+          - Inter-MediumItalic.ttf
+          - Inter-Regular.ttf
+          - Inter-Semibold.ttf
+          - Inter-SemiboldItalic.ttf
+          - Inter-Thin.ttf
+          - Inter-ThinItalic.ttf
```
Comment (sbihel) [inline project.yml:67]: I would suggest adding `UIUserInterfaceStyle: "Light"` to force light theme because it doesn't look great with dark mode

**Review by sbihel — APPROVED** [review]

File: project.yml (hunk 5 above)
Comment (Juliano1612) [reply project.yml:67]: good catch!

File: Targets/AppUIKit/Sources/HomeView.swift (hunk 4 above)
Comment (Juliano1612) [reply Targets/AppUIKit/Sources/HomeView.swift:8]: Yes, it is. The reference design doesn't have the tabs to separate wallet and verifier, so I added this just to make it easier to switch. I don't think we should go with tabs as a definite solution

**Pull request merged**